common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -53,7 +54,8 @@ case class ActivationMessage(
     activationId: ActivationId,
     activationNamespace: EntityPath,
     content: Option[JsObject],
-    cause: Option[ActivationId] = None)
+    cause: Option[ActivationId] = None,
+    traceMetadata: Option[SpanMetadata] = None)
```

Comment:
If the system has issues currently with the payload size sent to the actions by the clients, then that's probably a separate conversation to have. A design inspired from HTTP that sends the `headers`/`metadata` first, then streams the content directly to the action, might be a less limiting one.

For this particular PR with tracing we're probably worrying about a tiny slice of the current payload limit.  We're probably talking about 1-4KB to be generous ? If we move this information elsewhere, then we introduce latency b/c the controller and invoker need to do an outbound request to get this info, then I'm not sure of the benefits of tracing anymore, provided it impacts latency.